hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BaseLoadBalancer.java

```diff
@@ -1335,6 +1236,20 @@ protected final boolean idleRegionServerExist(Cluster c){
     return isServerExistsWithMoreRegions && isServerExistsWithZeroRegions;
   }
 
+  /**
+   * Assign system regions to master in maintenanceMode
+   */
+  @NonNull
+  protected final Map<ServerName, List<RegionInfo>>
+    assignSystemRegionsToMaster(Collection<RegionInfo> regions, List<ServerName> servers) {
```

Comment:
maintenanceMode has special balancer, so this method is no used?

Reply:
Oh, Thanks. Should be a rebase issue. Let me remove this method.